<launch>   

    <node name="robot_controller" pkg="robot_navigator" type="robot_controller">
        <param name="cmd_vel_topic" type="string" value="/cmd_vel" />
        <!-- <param name="cmd_vel_topic" type="string" value="/mobile_base/commands/velocity" /> -->
        <param name="odom_topic" type="string" value="/odom" />
        <param name="base_link" type="string" value="base_footprint" />
        <param name="odom_link" type="string" value="odom" />
        <param name="topic_octomap" type="string" value="/octomap_point_cloud_centers" />
        <param name="robot_dimention_width" type="string" value="0.5" />
        <param name="robot_dimention_height" type="string" value="0.5" />
        
    </node>
    
</launch>